http/lbtransport/policy_test.go

```diff
@@ -40,75 +37,70 @@ func TestRoundRobinPolicy_PickWithBlacklist(t *testing.T) {
 		},
 	}
 
+	// Test Global blacklist.
+	picker := rr.Picker()
```

Comment:
maybe make these things into functions? I.e. something like assertTargetsReturnedInOreder(picker, numCalls=3, targets[1], targets[2], targets[0]) and the same for stuff below?